test-tools/addr2line/addr2line.py

```diff
@@ -145,6 +227,9 @@ def main():
     parser.add_argument("--wabt", type=Path, help="path to wabt")
     parser.add_argument("--wasm-file", type=Path, help="path to wasm file")
     parser.add_argument("call_stack_file", type=Path, help="path to a call stack file")
+    parser.add_argument(
+        "--no-addr", action="store_true", help="use call stack without addresses"
```

Comment:
since wamr outputs *addresses-like info* (actually offset to the pre-compiled function start) in call stack in fast-interpreter mode, how about 
```suggestion
        "--no-addr", action="store_true", help="use call stack without addresses or from fast-interpreter mode"
```